### PR TITLE
Creating sandbox to preview blogs

### PR DESCRIPTION
**Pull request by Garrett1Tolbert**

### CODE REVIEW

**Comment by blaccsmith** [comment]
Socials (twitter, github, etc.) don't show on mobile. Let's find a place to put them in the `BlogHeader` component

**Comment by blaccsmith** [comment]
@cawrites Can you check this out? This is a good start but am curious if it can be better?

**Comment by cawrites** [comment]
@Garrett1Tolbert are you looking for feedback on layout or content?

**Comment by blaccsmith** [comment]
> @blaccsmith are you looking for feedback on layout or content?

Both. Looking for whatever feedback you have!

File: .github/BLOG_CONTRIBUTING.md [hunk 1]
```diff
@@ -10,35 +10,47 @@ Run `yarn` to install dependencies and then `yarn dev` to start the server. Once
 
 ## Writing
 
-Go to the `blogs` folder at the root of the project and you should see `example.mdx`. [MDX](https://mdxjs.com/) is like Markdown, but lets us add custom components to our Markdown to make it special and our own!
+Go to the `blogs` folder at the root of the project and you should see an `example.mdx` file. This file shows an example for you to reference when creating yoour blog. We're using [MDX](https://mdxjs.com/) for content creation — it's like Markdown, but lets us add custom components to our Markdown to make it special and our own!
```
Comment (DarrylBrooks97) [inline .github/BLOG_CONTRIBUTING.md:13]: Added an extra 'o' in 'your'

**Review by DarrylBrooks97 — APPROVED** [review]
LGTM

**Pull request merged**